data/issues.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+import requests
+
+from data.models import (
+    Issue,
+    Label,
+    )
+from data.newcomers import active_newcomers
+from data.models import Contributor
+from data.webservices import webservices_url
+from data.convert import convert
+
+
+def fetch_issues(hoster):
+    """
+    Get issues opened by newcomers
+
+    :param hoster: a string representing hoster, e.g. 'github'
+    :return: a json of issues data
+    """
+    logger = logging.getLogger(__name__)
+    hoster = hoster.lower()
+    import_url = webservices_url('issues/%s/all' % hoster)
+
+    headers = {'Content-Type': 'application/json'}
+    try:
+        response = requests.get(
+            url=import_url,
+            headers=headers,
+        )
+        response.raise_for_status()
+    except Exception as e:
+        logger.error(e)
+        return
+    issues = response.json()
+
+    # Removing issues which are not opened by newcomers
+    issues_list = []
+    for issue in issues:
+        if issue['author'] in active_newcomers():
+            issues_list.append(issue)
+    return issues_list
+
+
+def import_issue(hoster, issue):
+    """
+    Import issue data to database
+
+    :param hoster: a string representing hoster
+    :param issue: a dict containing issue's data
+    """
+    logger = logging.getLogger(__name__)
+    number = issue.get('number')
+    assignees = issue.pop('assignees')
+    labels = issue.pop('labels')
+    author = issue.pop('author')
+    issue['created_at'] = convert(issue['created_at'])
+    issue['updated_at'] = convert(issue['updated_at'])
+    try:
+        author, created = Contributor.objects.get_or_create(login=author)
+        issue['author'] = author
+        issue['hoster'] = hoster
+        issue_object, created = Issue.objects.get_or_create(**issue)
+        # Saving assignees
+        assignees_list = []
+        for assignee in assignees:
+            assignee, created = Contributor.objects.get_or_create(
```

Comment:
it is confusing to redefine a variable in a loop.  how are the two variables different?  describe them in the variable name.

oh frig.  you did do this below for `label` and `label_object` - why didnt you do it here?!